wikipron/extract/default.py

```diff
@@ -0,0 +1,19 @@
+"""Default word and pron extraction."""
+
+import itertools
+
+from wikipron.extract.core import yield_pron
+
+
+IPA_XPATH = '//span[@class = "IPA"]'
+
+
+def _yield_phn(request, config):
+    for pron_xpath in request.html.xpath(config.pron_xpath_selector):
+        yield from yield_pron(pron_xpath, IPA_XPATH, config)
+
+
+def extract_word_pron_default(word, request, config):
+    words = itertools.repeat(config.casefold(word))
+    prons = _yield_phn(request, config)
+    yield from zip(words, prons)
```

Comment:
With only Khmer as a non-default case, I'm not entirely sure if this is the right level of abstraction for various things. For this PR, I'm happy with the way it is. (I do not plan to handle other special languages within this PR.) When we add more languages under `wikipron/extract/`, we may have some refactoring here.